### DISCUSSION TITLE
Is it possible to use OR or ORFS and integrate with Google Circuit Training?

### DISCUSSION THREAD

**Discussion by RaidToRadar** [The-OpenROAD-Project/OpenROAD#3481] [General]
I have been wondering if there is a flow that allows the connection between OR or ORFS and be able to use the synthesized/placed .def files to be used as an input for the Deep Reinforcement Learning placement algorithm created by Google (Circuit Training)?

**Comment by maliberty** [comment]
I don't know of such a flow but I think you could build one.
